CONTRIBUTORS.yml

```diff
@@ -25,3 +25,4 @@
 - ashleyjagai
 - Wisesofthemall
 - kkellybonilla
+- ciaracade
```

Comment:
It looks like your editor settings may be removing the final newline - would you mind enabling the `EditorConfig for VS Code` extension in your workspace so that it respects the `.editorconfig` configuration? And then resave/recommit this file?

Reply:
should be fixed now